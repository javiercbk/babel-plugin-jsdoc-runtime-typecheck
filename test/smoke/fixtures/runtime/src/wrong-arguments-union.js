// @typecheck

/**
 * @param {{id: Number, name: String}|null} data
 * @returns {Number}
 */
function test(data) {
    if (data) {
        return data.id;
    }
}

test({id: '2'});
