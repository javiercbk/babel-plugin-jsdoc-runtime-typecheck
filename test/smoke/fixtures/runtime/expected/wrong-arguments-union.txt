'TypeError: \n    Parameter "data" in function "test" has wrong type.\n    Expected type: {"id":"Number","name":"String"}|null\n    Current value: "{"id":"2"}"\n'
